supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -13,109 +904,211 @@
 *See also*: xref:jboss-eap[JBoss EAP]
 
 [discrete]
-[[jboss-eap]]
-==== JBoss EAP (noun)
-*Description*: JBoss EAP is the approved shortened form of xref:red-hat-jboss-enterprise-application-platform[Red Hat JBoss Enterprise Application Platform].
+[[remoting]]
+==== remoting (noun)
+*Description*: The "remoting" subsystem is used to configure inbound and outbound connections for local and remote servers. Write in lower case. See xref:remoting-heading[Remoting] entry for the correct usage in headings.
 
 *Use it*: yes
 
-*Incorrect forms*: EAP, JBoss
+*Incorrect forms*: Remoting
 
-*See also*: xref:red-hat-jboss-enterprise-application-platform[Red Hat JBoss Enterprise Application Platform]
+*See also*: xref:remoting-heading[Remoting]
+
+[discrete]
+[[remoting-heading]]
+==== Remoting (noun)
+*Description*: Use "Remoting" when referring to the xref:remoting[remoting] subsystem in titles and headings. Capitalize the word. See xref:remoting[remoting] entry for the correct usage in general text.
 
+*Use it*: yes
+
+*Incorrect forms*: remoting
+
+*See also*: xref:remoting[remoting]
 
 [discrete]
-[[activemq]]
-==== ActiveMQ (noun)
-*Description*: ActiveMQ should not be used by itself when referring to the built-in messaging in JBoss EAP.
+[[repo]]
```

Comment:
This entry is of general interest beyond this specific product. Move to the main glossary.

Reply:
I've removed this entry and will add it to the main glossary as part of a separate pull request in due course.

Reply:
Entry appears to still be in the EAP doc.

Reply:
Hi @mportman12. The ActiveMQ entry was removed as part of the second commit performed yesterday at https://github.com/redhat-documentation/supplementary-style-guide/pull/97/commits/62bdbb168ae66ee6fbf1ecb03faa969e79b4fd15.

Reply:
Apologies for the confusion. I meant that I thought `repo`, not `ActiveMQ`, belonged in the main glossary.

Reply:
Hi @mportman12. Ok no problem. I will remove the `repo` entry from jboss_eap.adoc.

Just to clarify. Did you also mean to instruct removal of the `ActiveMQ` entry or should I leave that in jboss_eap.adoc?

Reply:
I don't know enough about ActiveMQ to say either way. If you think it's a JBoss-specific term, I trust your judgement!

Reply:
Hi @mportman12. I've just pushed a third commit that removes the `repo` entry and reinstates the `ActiveMQ` entry.